README.md

```diff
@@ -0,0 +1 @@
+Deluge is a lightweight, Free Software, cross-platform BitTorrent client. See: https://deluge-torrent.org/
```

Comment:
No need to have it here.

Reply:
Removed Readme.md as it is not required for flatpak building and packaging.